### ISSUE TITLE
2.1.0 Build errors

### ISSUE THREAD

**Issue by keithZmudzinski** [micnil/vscode-diff#3]
Hi,

After updating to 2.1.0, our project started encountering build errors.
Pinning to 2.0.2 resolves them, so we're not blocked, but I wanted to bring it up

Here's one example: 

```
ERROR in ./node_modules/vscode-diff/dist/vs/base/common/diff/diff.js 4:241-284
Module not found: Error: Can't resolve './vs/base/common/diff/diffChange' in '/Users/<path-to-project>/node_modules/vscode-diff/dist/vs/base/common/diff'
 @ ./node_modules/vscode-diff/dist/index.js 1:701-738
 @ ../../../../../private/tmp/broccoli-51669SBRmBJVjMJ5p/cache-616-webpack_bundler_ember_auto_import_webpack/app.cjs 75:45-67
```
This is with `npm` version `9.5.0`
and `node` version `v18.15.0`

If there's any other info you'd like, please just give me a ping!

**Comment by micnil** [comment]
Thanks for reporting so quickly! This should be fixed in v2.1.1.

**Comment by keithZmudzinski** [comment]
Awesome, thank you very much
